application.py

```diff
@@ -128,6 +159,59 @@ def handle_invalid_usage(error):
     response.status_code = error.status_code
     return response
 
+"""CUSTOM ROUTE: Sparse HAC"""
+@application.route("/useclustersparse/", methods=["POST"])
+def handleUSECluster():
+    print('This is Sparse HAC!')
```

Comment:
Hi Jeff, thank you so much for this insight! This makes sense :D

So in the application.py, I have replaced some useful print statements with application.logger.debug
In the use_cluster.py file, since we don't have the application initialized, I have used Python's logging module and converted all useful print statements into logging.debug() ones

I have made sure to delete all extra print statements 👍